### ISSUE TITLE
Feature request: Swipe between groups

### ISSUE THREAD

**Issue by TheLastProject** [CatimaLoyalty/Android#2129]
### Discussed in https://github.com/CatimaLoyalty/Android/discussions/2128

<div type='discussions-op-text'>

<sup>Originally posted by **NtWriteCode** October 23, 2024</sup>
Right now I'm using the latest available from Google Play, so not sure if it's already a feature.

Right now the only way to switch between groups is clicking its names. It would be amazing to have a feature, where I could swipe left or right to change between the groups :)</div>

**Comment by TheLastProject** [comment]
Sorry, ignore my last comment, I got confused with another issue.

We used to have this feature but it got removed because it triggered too often and people got bothered.

Related issues:
- https://github.com/CatimaLoyalty/Android/issues/85
- https://github.com/CatimaLoyalty/Android/issues/699

I'm okay with this being re-added if we find a way to make it not trigger too much.

Important parts for the implementation:
1. It must be clear the swipe is about to happen
2. The sensitivity should be of such a value that it's fine for everyone by default, please not more and more settings in the setting menu :(

**Comment by NtWriteCode** [comment]
I totally agree, it should only be added if we can distinguish reliable between scrolling and swiping :) Sadly I have 0 knowledge on Android development, hence I can't really help with it :)

**Comment by Methaneus** [comment]
Going to give this a bump; it's a small annoyance to me that I can't swipe through the categories.
